### DISCUSSION TITLE
SVG export with interactions

### DISCUSSION THREAD

**Discussion by shlomi-dr** [freeplane/freeplane#1112] [General]
Hi,

I absolutely love freeplane! It is, however, rather difficult to share maps with other folks in today's "everything-is-saas" model. Also, none of the "paid mindmap saas" that can read freeplane format are actually useable..

I looked around and found a [clone repo](https://github.com/c-illy/freeplane-svggen-interaction) (which seems to not have made any modifications), but at least its description is useful:

> Add (javascript) interaction to svg exports from freeplane. Useful to share foldable mind maps with many people directly in their web browser. Main feature : branch folding. Minor features : auto scroll to center during page load, appropriate standard mouse cursors.

This would be absolutely fantastic, as I could share and present SVGs on any machine and to anyone. Without abilities to fold, it is very difficult to present larger topics and keep the audience focused.

Would love to get the thoughts on this,

Thanks!

**Comment by cadoitrouler** [comment]
Hi shlomi-dr,
Yes you are right, would be great this kind of feature.
I am waiting for years for an easy way in Freeplane to being able to export a map as an html page, only the map itself with interaction possible (folding, unfolding of branches etc.), as that was the case with java applets before, i.e. something which is not usable any more for years now on any browser.
You can do that with draw.io for instance (so easy). But of course Freeplane is much more straightforward for conceiving maps, but unfortunately impossible to publish my maps correctly on my website.
I tried hard with the Java web start thing (jnlp), did not work at all on my website pages, maybe I did something wrong, if someone could give a procedure which would work eventually, would be nice (a precise one, not just do sort of like this or that and you have to invent the rest).
How could someone imagine using even such a nice and complete mindmapping soft as Freeplane if it lacks the basic ability to easily publish a map in its interactive form exactly as it has been built up in the soft into a web page (integration)??!
I think this is high priority, before any other cosmetic improvement, and the day it will be possible, really I will open a bottle of champagne!
Thanks if someone will pay even the slightest attention to this message and, FINALLY, implement this extremely long awaited possibility.
